src/pages/team.en.tsx

```diff
@@ -23,9 +24,14 @@ type MemberCardInfo = {
 
 const MemberCard: React.FC<MemberCardInfo> = (props) => (
   <div
-    className={`jf-member-card column is-9 is-12-touch px-6 py-9 py-6-touch  ${props.className}`}
+    className={classnames(
+      "jf-member-card column",
+      "is-flex is-flex-direction-column",
+      "is-9 px-6 py-9 py-6-mobile",
```

Comment:
just to check, the `is-12-touch` class is dropped intentionally here?

Reply:
yes, we decided to use the desktop layout for tablet and after taking that out the mobile layout is handled by default bulma column behavior. Thanks for checking!